[Feature-8030][docs] Add sqoop task doc (#12855)
apache/dolphinscheduler: docs/docs/en/guide/task/sqoop.md

# Sqoop Node

## Overview

> Sqoop task type for executing Sqoop application. The workers run `sqoop` to execute  sqoop tasks.

Reviewer: ```suggestion
Sqoop task type for executing Sqoop application. The workers run `sqoop` to execute sqoop tasks.
```

## Task Example

This example demonstrates importing data from MySQL into Hive. The MySQL database name is `test` and the table name is `example`. The following figure shows sample data.

![sqoop_task01](../../../../img/tasks/demo/sqoop_task01.png)

### Configuring the Sqoop environment

If you are using the Sqoop task type in a production environment, you must ensure that the worker can execute the `sqoop` command.

### Configuring Sqoop Task Node

you can configure the node content by following the steps in the diagram below.

![sqoop_task02](../../../../img/tasks/demo/sqoop_task02.png)

The key configuration in this sample is shown in the following table.

| **Parameter**                       | **Value**                                                         |
|-------------------------------------|-------------------------------------------------------------------|
| Job Name                            | sqoop_mysql_to_hive_test                                          |
| Data Source - Type                  | MYSQL                                                             |

> | Data Source - Datasource            | MYSQL MyTestMySQL(MyTestMySQL is the name of my test data source) |

Reviewer: ```suggestion
| Data Source - Datasource            | MYSQL MyTestMySQL(You could change MyTestMySQL to the name you like) |
```
